IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py

```diff
@@ -1058,6 +1058,19 @@
     Z0 = cms.double(0.131175)
 )
 
+# From 2023 PbPb data run 374719 
+Realistic2023PbPbCollisionVtxSmearingParameters = cms.PSet(
+    Phi = cms.double(0.0),
+    BetaStar = cms.double(50),
+    Emittance = cms.double(2.055e-08),
```

Comment:
```suggestion
    Emittance = cms.double(5.854e-08),
```
Based on discussion at https://github.com/cms-sw/cmssw/pull/43217/files#r1388231420